tests/src/scope/scope.ts

```diff
@@ -0,0 +1,69 @@
+import assert from "assert";
+import * as svelte from "../../../src";
+import { FlatESLint } from 'eslint/use-at-your-own-risk';
```

Comment:
Could you change it to use the `Linter` class instead of `FlatESLint`? I believe that using the `Linter` class is sufficient for this test.

https://eslint.org/docs/latest/integrate/nodejs-api#linter